test/suite/functional/Generator/SchemaGeneratorTest.php

```diff
@@ -40,12 +40,60 @@ public function exampleProvider(): array
                 self::BASE_NAMESPACE . SchemaGenerator::NAMESPACE_SUBPATH . '\\ItemMandatoryObject',
                 ConfigurationBuilder::fake()->build(),
             ],
-            'All of'       => [
+            'All of'          => [
                 '/Schema/extendedItem.yaml',
                 '/Schema/AllOfItem.php',
                 self::BASE_NAMESPACE . SchemaGenerator::NAMESPACE_SUBPATH . '\\ExtendedItem',
                 ConfigurationBuilder::fake()->build(),
             ],
+            'One of - Dog'    => [
```

Comment:
do you think it's really needed to add these here? I mean, with allOf it does make sense because the output schema object is a merger of N openapi schemas, but here with oneOf it doesn't really change a thing from any other regular schema

Reply:
only this one test/suite/functional/Generator/Schema/GetExampleResponseBody.php is worth checking IMO, and I don't see it being checked